code/modules/mob/living/simple_animal/abnormality/he/road_home.dm

```diff
@@ -0,0 +1,387 @@
+/mob/living/simple_animal/hostile/abnormality/road_home
+	name = "The Road Home"
+	desc = "An abnormality ressembling a small girl."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "road_home"
+	icon_living = "road_home"
+	maxHealth = 1300
+	health = 1300
+	move_resist = MOVE_FORCE_STRONG //So she can't be yeeted away and delayed indefinitely
+	move_to_delay = 10 //She needs to be slow so she doesn't reach home too fast
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0, WHITE_DAMAGE = 2, BLACK_DAMAGE = 2, PALE_DAMAGE = 2) //Immune to red because catt mentions physical attacks can't hurt her at all.
```

Comment:
I highly recommend increasing red to 0.2 or 0.3
It's not a WAW and has softlocked a couple rounds so far.